utils/mock-data-loader/gef/eligibilityCriteriaAmendments.js

```diff
@@ -0,0 +1,43 @@
+const { FACILITY_TYPE } = require('@ukef/dtfs2-common');
+
+const ELIGIBILITY_CRITERIA_AMENDMENTS = [
+  {
+    version: 1,
```

Comment:
`libs/common` constant please.

Reply:
As discussed offline will leave this for now, as I don't think the version numbers will be used in the code much